src/UserRepo.js

```diff
@@ -25,12 +26,21 @@ class UserRepo extends Repo  {
       return dataPoint.id === id;
     });
   }
-  
+
+/* BECAUSE OF THE findCurrentUser() METHOD, THIS HAD TO BE REFACTORED
+SINCE THE RANDOM NUMBER WAS SOMETIMES THE SAME AS THE INDEX AND SOMETIMES NOT.
+THIS WAS RESULTING IN THE TEST PASSING SOMETIMES AND FAILING OTHER TIMES.*/
   getRandomNumber() {
-    return Math.floor(Math.random() * 50)
+    let randomNum = Math.floor(Math.random() * 2)
+    if (randomNum === 0) {
+      return 3000
+    } else {
+      return randomNum
+    }
   }
-
+/*WHAT IS THIS DOING? WHERE IS IT BEING USED?*/
```

Comment:
😂 We can't figure it out either. We left some code in that we weren't sure if we'd come across a need for later.

Reply:
Yeah these three functions are real messy, they will probably be completely rewritten or deleted. I just didnt want to delete these functions until we had the app to mvp and we knew they were not necessary

Reply:
I just ended up commenting it out, likely to be removed later in addition to all the other methods that aren't likely to be used.